### PR TITLE
Add image source to alert with type function

### PR DESCRIPTION
**Pull request by lk321**
It's a simple change 👷 

This change is to put a custom image source in the DropDownAlert menu.

Example: 
```ssh 
myDropDown.alertWithType('info','Test title', 'message', null, require('assets/images/simple-logo.png'));

```
Result:
image

### CODE REVIEW

**Comment by testshallpass** [comment]
@lk321 Please describe more on the use case. Thanks.

**Comment by lk321** [comment]
@testshallpass done ;)

**Comment by testshallpass** [comment]
@lk321 Technically, you could use prop `infoImageSrc` to acheive this.

**Comment by booboothefool** [comment]
I believe I could use this feature. I have one <DropdownAlert /> at the root of my app, however I would like to be able to pass dynamic images into it e.g. different user avatars.

**Comment by booboothefool** [comment]
@testshallpass I believe I could use this feature. I have one <DropdownAlert /> at the root of my app, however I would like to be able to pass dynamic images into it e.g. different user avatars.

File: imageview.js [hunk 1]
```diff
@@ -29,6 +29,9 @@ export default class ImageView extends Component {
       if (!style['height']) {
         style['height'] = DEFAULT_IMAGE_DIMENSIONS;
       }
+      if(isRemote) {
+        style['borderRadius'] = (style['width'] / 2);
```
Comment (testshallpass) [inline imageview.js]: this borderRadius should be opt-in setting

File: DropdownAlert.js [hunk 2]
```diff
@@ -437,20 +441,20 @@ export default class DropdownAlert extends Component {
...
-    return <Label {...messageTextProps} style={StyleSheet.flatten(messageStyle)} numberOfLines={messageNumOfLines} text={this.state.message} />;
+    return <Label {...messageTextProps} style={[StyleSheet.flatten(messageStyle), StyleSheet.flatten(this.state.payload.messageStyle || {})]} numberOfLines={messageNumOfLines} text={this.state.message} />;
   }
   render() {
-    const { isOpen, type } = this.state;
+    const { isOpen, type, payload } = this.state;
     if (isOpen) {
       let style = this.getStyleForType(type);
-      const source = this.getSourceForType(type);
+      const source = payload.customSource ? payload.customSource : this.getSourceForType(type);
```
Comment (testshallpass) [inline DropdownAlert.js]: call it `payload.imageSrc`

**Comment by testshallpass** [comment]
@lk321 Closing, I reworked it in PR #200

**Pull request closed without merging**